### DISCUSSION TITLE
TypeError: seeder is not a constructor

### DISCUSSION THREAD

**Discussion by WOOSERK** [tada5hi/typeorm-extension#409] [Q&A] [answered]
Hi. First of all, thank you for making such a good library.
But I need to help.

When I enter command:
`ts-node ./node_modules/typeorm-extension/dist/cli/index.js `

I am getting following error:
```
TypeError: seeder is not a constructor
    at C:\Users\user\Desktop\boostcamp-membership\weview\server\node_modules\typeorm-extension\src\seeder\module.ts:119:19
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\user\Desktop\boostcamp-membership\weview\server\node_modules\typeorm-extension\dist\seeder\module.js:5:58)
```

the data-source file is located at:
`/src/database`

Seeds and factory are respectively located under:
`src/database/seeds/`
and
`src/database/factories/`

How to fix it?

**Comment by bidossessi** [comment, marked as answer]
Possibly related to #498.
